app/controllers/auth/registrations_controller.rb

```diff
@@ -10,4 +12,11 @@ def new
     @have_users = User.any?
     super
   end
+
+  # Restrict the amount of admin users to 1, see bsc#1040831
+  def restrict_admin_users
+    return false if User.count.zero?
+    flash[:alert] = "Admin user already exists."
+    redirect_to root_path
```

Comment:
app/controllers/auth/registrations_controller.rb:18:5: C: Use a guard clause instead of wrapping the code inside a conditional expression.
    unless User.count.zero?

:-(